src/workflow/async.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ASYNC_H
+#define ASYNC_H
+#include <stdbool.h>
+#include <stdint.h>
+
+enum ready {
+    Ready,
+    NotReady,
+};
+
+bool workflow_async_busy_get(void);
```

Comment:
please add docstrings to any new .h functions

Reply:
ack